specification/dimensions/provider.md

```diff
@@ -0,0 +1,32 @@
+# Provider
+
+A Provider is an entity that made the infrastructure and/or services available for purchase. It is commonly used for cost analysis and reporting scenarios.
+
+The Provider column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values.
+
+See [Appendix: Origination of the cost data](../appendix/origination_of_cost_data.md) section for context (Provider/Publisher/Invoicing Entity) in various purchasing scenarios.
+
+## Column ID
+
+Provider
+
+## Display name
+
+Provider
+
+## Description
+
+The entity that made the infrastructure and/or services available for purchase.
```

Comment:
Nit: Should be resources and/or services here and elsewhere (you may have already said you're making that change)

Reply:
replaced in all 3 branches (infrastructure and/or services -> resources and/or services)